mongo/data_store.go

```diff
@@ -350,3 +411,15 @@ func modifyJobs(jobs []models.Job, offset int, limit int) []models.Job {
 	}
 	return modifiedJobs
 }
+
+// modifyTasks takes a slice, of all the tasks in the data store, determined by the offset and limit values
+func modifyTasks(tasks []models.Task, offset int, limit int) []models.Task {
```

Comment:
This wont be needed if we rely on mongo to give us the correct tasks back, see earlier comment

Reply:
removed now